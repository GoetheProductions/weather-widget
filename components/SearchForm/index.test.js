import React from 'react';
import renderer from 'react-test-renderer';
import Component from './index';

const component = renderer.create(
  <Component searchQuery="" handleSubmit={() => {}} handleSearch={() => {}} />,
);

test('Form matches snapshot', () => {
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('Should be form', () => {
  const tree = component.toJSON();
  expect(tree.type).toBe('form')
});
