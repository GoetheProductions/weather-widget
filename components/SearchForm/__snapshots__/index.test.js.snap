// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form matches snapshot 1`] = `
<form
  className="form-inline"
  onSubmit={[Function]}
>
  <div
    className="form-group"
  >
    <input
      className="form-control"
      onChange={[Function]}
      placeholder="City"
      type="text"
      value=""
    />
  </div>
  <button
    className="btn btn-default"
    disabled={true}
    type="submit"
  >
    Search
  </button>
</form>
`;
